### ISSUE TITLE
Program groups minimise to little window

### ISSUE THREAD

**Issue by brannanz** [freedom7341/Progman-x64#24]
Minimised program groups appear as squished windows instead of icons. Would there be any way to restore the Windows 3.1 functionality? You can actually see the same behaviour when minimising other windows without Explorer running, so I imagine Microsoft changed some kind of functionality.

**Comment by freedom7341** [comment]
Thought about implementing this at some point but wasn't worth the hassle to me considering this project is mostly MS code. Basically windows are drawn by GDI (or whatever subsystem is used now) and in 95/NT4 they were changed to render minimized windows as shrunk titlebars instead of as their icons. Only way to restore this is to write a program that creates a "taskbar" that mimics the 3.1 functionality or somehow hook into the window drawing code and replicate the icon drawing code.

**Comment by TheColorOfFear** [comment]
I feel like this is essential to having a reasonably intuitive shell though, given that some programs (ex. Steam (the black rectangle ) or Microsoft Teams (the white rectangle ) seen on the left of the screen in the attached image ) minimize to often un-movable mini-windows that sometimes can only be restored with alt-tabbing into them ![image](https://github.com/freedom7341/Progman-x64/assets/68704661/5f483102-3649-449c-a855-029fe89b8f91)

**Comment by freedom7341** [comment]
Yeah it'd be nice to fix that, steam and chrome and other electron programs all seem to not play well with their window borders. I don't exactly want to write my own taskbar at the moment though. I'm working on an entirely new shell and if anyone wants to take this up I'm willing to help them integrate it into Progman or maybe make it a separate program?

**Comment by freedom7341** [comment]
I won't totally discount this. I might rewrite parts of progman and fix some other bugs along the way, while I'm at it I'll try to implement this as well.
